src/js/main.js

```diff
@@ -42,8 +42,8 @@ var CRUMINA = {};
                 "tolerance": 5,
                 "classes": {
                     "initial": "animated",
-                    "pinned": "swingInX",
-                    "unpinned": "swingOutX"
+                    // "pinned": "swingInX",
```

Comment:
If we decide to leave commented out code, we need to include a reason as to why.
Ex: if we need to leave the commented lines because the change is temporary and we might have to revert to the header hiding, it needs to be explained by adding a comment in the code.